### PR TITLE
New package: vulkan-tools-1.1.77.0

### PR DESCRIPTION
**Pull request by 5nefarious**
Includes `cube`, `cubepp`, and `vulkaninfo`.

### CODE REVIEW

File: srcpkgs/vulkan-headers/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+revision=1
+build_style=cmake
+wrksrc=${_pkgname}-sdk-${version}
+short_desc="Vulkan header files"
+maintainer="Arvin Ignaci <arvin.ignaci@gmail.com>"
+license="Apache-2.0"
+homepage="https://www.khronos.org/vulkan/"
+distfiles="https://github.com/KhronosGroup/${_pkgname}/archive/sdk-${version}.tar.gz"
+checksum=b2f532bfd1d8e7594f131a4aa79358bfe4fd0aa59d3292dbafd484223d56ef16
+
```
Comment (maxice8) [inline srcpkgs/vulkan-headers/template]: extra blank line

File: srcpkgs/vulkan-headers/template [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'vulkan-headers'
+pkgname=vulkan-headers
+_pkgname=Vulkan-Headers
```
Comment (maxice8) [inline srcpkgs/vulkan-headers/template]: it is a new package, i'd vote for using the proper upstream name.

File: srcpkgs/vulkan-tools/template [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+hostmakedepends="python3 pkg-config"
+makedepends="vulkan-headers vulkan-loader libxcb-devel libxkbcommon-devel
+ wayland-devel libwayland-egl libXrandr-devel"
+short_desc="Official Vulkan tools and utilities"
+maintainer="Arvin Ignaci <arvin.ignaci@gmail.com>"
+license="Apache-2.0"
+homepage="https://www.khronos.org/vulkan/"
+distfiles="https://github.com/KhronosGroup/${_pkgname}/archive/sdk-${version}.tar.gz"
+checksum=15eaefa9d9370c58aac3458b3cb67909759fb4e965d737a57e7de6cc108fabc6
+
```
Comment (maxice8) [inline srcpkgs/vulkan-tools/template]: extra blank line

File: srcpkgs/vulkan-tools/template [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'vulkan-tools'
+pkgname=vulkan-tools
+_pkgname=Vulkan-Tools
```
Comment (maxice8) [inline srcpkgs/vulkan-tools/template]: it is a new package, i vote to use the proper upstream name

**Comment by 5nefarious** [comment]
Status update: I know I haven't made any changes on this branch in a while. I am still having trouble getting the i686 build working. I'll probably push some changes once that issue is resolved.

It seems likely that this package will have to be restricted to i686 and x86_64, as everything but the x86_64 build is currently failing. 32-bit support should be possible, though it is not officially supported.

**Pull request closed without merging**